# no input for this service
---
SceneOcclusion data